stable/prometheus-adapter/values.yaml

```diff
@@ -16,6 +16,7 @@ nodeSelector: {}
 prometheus:
   url: http://prometheus.default.svc
   port: 9090
+  path:
```

Comment:
shouldn't this be /prometheus by default?

Reply:
@davidkarlsen Nope, by default prometheus serves / URI. There is an option to add some prefix for Prometheus URIs and make it serve /prefix/. This is used in environments where you want to share same hostname for multiple services and route HTTP requests based on URI path via API gateway.